Marlin/src/lcd/language/language_en.h

```diff
@@ -350,6 +350,13 @@ namespace Language_en {
   LSTR MSG_PID_BAD_EXTRUDER_NUM           = _UxGT("Autotune failed. Bad extruder.");
   LSTR MSG_PID_TEMP_TOO_HIGH              = _UxGT("Autotune failed. Temperature too high.");
   LSTR MSG_PID_TIMEOUT                    = _UxGT("Autotune failed! Timeout.");
+  LSTR MSG_MPC_MEASURING_AMBIENT          = _UxGT("Measuring ambient heatloss");
```

Comment:
Some users will only see "Measuring ambient he" unless scrolling status message is enabled, so LCD messages should be shorter.